lib/get-token-header.js

```diff
@@ -0,0 +1,8 @@
+export default (token) => {
+    switch(true) {
+        case token.startsWith("glcbt-"):
+          return "JOB-TOKEN"
+        default:
+          return "PRIVATE-TOKEN"
+      }
```

Comment:
```suggestion
    if (token.startsWith("glcbt-")) return "JOB-TOKEN";
    return "PRIVATE-TOKEN";
```

I don't really understand the need to use `switch(true)` here?

Reply:
@fgreinacher: It's more complicated to support... We would have to add configuration parameters to support this. Currently, there is no regression with the change since if the prefix has been modified, it will continue to fall into the default case with it's the current implementation. It might be necessary to add a note in the documentation that currently CI job tokens with custom prefixes are not supported. I hadn't really planned on making the changes to support custom prefixes.